README.md

```diff
@@ -0,0 +1,89 @@
+![Logo](logo.gif)
+[![Build Status](https://travis-ci.org/idealista/apache_exporter-role.svg?branch=master)](https://travis-ci.org/idealista/apache_exporter-role)
+
+# Prometheus Apache Exporter Ansible role
+
+This ansible role installs a Prometheus Apache Exporter in a debian environment.
+
+- [Getting Started](#getting-started)
+	- [Prerequisities](#prerequisities)
+	- [Installing](#installing)
+- [Usage](#usage)
+- [Testing](#testing)
+- [Built With](#built-with)
+- [Versioning](#versioning)
+- [Authors](#authors)
+- [License](#license)
+- [Contributing](#contributing)
+
+## Getting Started
+
+These instructions will get you a copy of the role for your ansible playbook. Once launched, it will install an [Prometheus Apache Exporter](https://github.com/Lusitaniae/apache_exporter) server in a Debian system.
+
+### Prerequisities
+
+Ansible 2.3.x.x version installed.
+Inventory destination should be a Debian environment.
+
+For testing purposes, [Molecule](https://molecule.readthedocs.io/) with [Vagrant](https://www.vagrantup.com/) as driver (with [landrush](https://github.com/vagrant-landrush/landrush) plugin) and [VirtualBox](https://www.virtualbox.org/) as provider.
+
+### Installing
+
+Create or add to your roles dependency file (e.g requirements.yml):
+
+```
+- src: idealista.apache_exporter-role
+  version: 1.0.0
+  name: apache_exporter
+```
+
+Install the role with ansible-galaxy command:
+
+```
+ansible-galaxy install -p roles -r requirements.yml -f
+```
+
+Use in a playbook:
+
+```
+---
+- hosts: someserver
+  roles:
+    - role: apache_exporter
+```
+
+## Usage
+
+Look to the [defaults](defaults/main.yml) properties file to see the possible configuration properties. To work properly apache module `mod_status` should be enabled and configure.
+
+## Testing
+
+```
+molecule test
+```
+
+## Built With
+
+![Ansible](https://img.shields.io/badge/ansible-2.2.1.0-green.svg)
```

Comment:
2.3, same as prerequisites version

Reply:
Also you could add:
![Molecule](https://img.shields.io/badge/molecule-1.25.0-green.svg)
![Testinfra](https://img.shields.io/badge/testinfra-1.9.0-green.svg)